controllers/cluster_controller.go

```diff
@@ -275,20 +320,45 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 		if len(replicas) != int(*sts.Spec.Replicas) {
 			reqLogger.Info("Hasn't explored all replicas in replicaset - requeue")
-			return ctrl.Result{RequeueAfter: time.Duration(4 * time.Second)}, err
+			return ctrl.Result{RequeueAfter: time.Duration(60 * time.Second)}, err
 		}
 
-		reqLogger.Info("All replicas in replicaset explored, sending editTopology request")
-		if _, err := topologyClient.EditTopology(replicas, sts.GetName()); err != nil {
+		reqLogger.Info("All replicas in replicaset explored, creating graphql struct")
+		if curReplicaset, err := topologyClient.GetReplicasetValues(replicas, sts.GetName()); err != nil {
 			reqLogger.Info("Failed to execute EditTopology")
-			return ctrl.Result{RequeueAfter: time.Duration(4 * time.Second)}, err
+			return ctrl.Result{RequeueAfter: time.Duration(60 * time.Second)}, err
+		} else {
+			replicasets = append(replicasets, curReplicaset)
+		}
+
+		reqLogger.Info("Successfully generated graphQL variables for replicaset: " + sts.GetName())
```

Comment:
This comment is not clear without reading the source codes. Please think about those who will read these logs. It won't be just developers.